Implementacija/savetovaliste/app/Views/postavi_pitanje.php

```diff
@@ -64,186 +69,184 @@
+                                <textarea style="color: #021B79; border-color: #021B79; font-size: 16px; resize: none; width: 100%;" name="TekstPitanja" id=""
+                                          cols="355" rows="10" placeholder=" Tekst pitanja"></textarea>
+                                <?php  if(!empty($porukaTekstPitanja)) echo "<br> <span style='color:red; font-size:16px'>$porukaTekstPitanja</span> <br>"; ?>  
                                 <br>
 
                                 <!-- JAVNO/PRIVATNO -->
-                                <div><input type="checkbox" id="anonimus" name="anonimus" value="Bike">
-                                    <label for="anonimus" style="color: #021B79; font-size: 16px; font-weight: bold; "> Želim anonimno da postavim
-                                        pitanje</label>
+                                <div>
+                                    <input type="checkbox" id="anonimus" name="anonimus" value="1">
+                                    <label for="anonimus" class="letters_dark_blue"> Želim anonimno da odgovorim na pitanje</label>
                                 </div>
                                 <br>
 
                                 <!-- KO ODGOVARA -->
                                 <h4>Na ovo pitanje će moći da odgovore &nbsp&nbsp</h4>
-                                <select style="color: #021B79; border-color: #021B79; font-size: 16px; " name="anwser">
-                                    <option value="">Svi korisnici</option>
-                                    <option value="">Samo psiholozi</option>
+                                <select style="color: #021B79; border-color: #021B79; font-size: 16px; " name="koOdgovaraNaPitanje">
+                                    <option value="Svi">Svi korisnici</option>
+                                    <option value="Psiholozi">Samo psiholozi</option>
                                 </select>
                                 <br>
                                 <br>
                                 <!-- DUGMAD -->
                                 <div>
-
+                                    <?php $referencaOdustani=site_url("$controller/"); ?>
                                     <div style="float: right;">
-                                        <button onclick="window.location.href = 'pocetna_stranica.html';" class="w3-button buttons">
-                                            Odustani</button> &nbsp
-                                        <button type="submit" class="w3-button buttons"><u onclick="alert('Pitanje je poslato!')" style="text-decoration: none; font-weight: normal;">Pošalji</u></button>
+                                        <button onclick="" class="w3-button buttons" style="font-weight: normal;"> 
+                                            <a class="nema_podvlacenja" href=' . '$referencaOdustani' . '>Odustani</a>
+                                        </button> &nbsp
```

Comment:
Mene klik na ovo dugme vodi na `http://localhost:8080/Korisnik/postaviPitanje?category=Nema&NaslovPitanja=&TekstPitanja=&koOdgovaraNaPitanje=Svi`.
Mislim da ima samo neki type koji možeš da mu dodeliš da bi brisao formu, kao što submit ima `type="submit"`

Reply:
Mislila sam da odustani znaci da ga vrati na pocetnu stranicu, a ne da mu obrise podatke iz forme